readme.md

```diff
@@ -8,6 +8,12 @@ Unfortunately these jokes only work if you git them.
 
 ---
 
+**Q:** What can you do if you cannot push your git changes?
+
+**A:** Push harder ;)
```

Comment:
just a riff on this one would be.
```
**Q:** What can you do if you cannot push your git changes?

**A:** (in old Obi-wan voice) Use the force

Reply:
Created a pull request on https://github.com/raux/dad-jokes with the update. https://github.com/raux/dad-jokes/pull/1

Reply:
How about just "Use the force, Luke"